app/controllers/hq/partners_controller.rb

```diff
@@ -3,7 +3,8 @@ class Hq::PartnersController < ApplicationController
   layout 'application'
 
   def index
-    @partners= Partner.all
+    #@partners = Partner.all.paginate(:page => params[:page], :per_page => 5)
+    @partners = Partner.paginate(:page => params[:page], :per_page => (params[:per_page]))
```

Comment:
Does this warrant a controller spec? Issuing `get` to `index` calls `Partner.paginate` with right params?